cmd/provider-list/converter.go

```diff
@@ -72,3 +82,72 @@ func getProviderAndServiceName(name string) []string {
 	}
 	return nil
 }
+
+type ConverterParameters struct {
```

Comment:
nit: We may consider breaking this struct into separate ConfigurationMeta, ConfigurationPackage, etc. converter structs.

Reply:
Done!